### ISSUE TITLE
Tab stops inside dialog bodies have clipped focus-rings

### ISSUE THREAD

**Issue by snowystinger** [adobe/spectrum-css#539]
## Description

If an input or something else that can get a focus ring is displayed in the body of a dialog and next to one of the edges of the body content it will be overflow:hidden and have about a pixel or so cut off.

## Steps to reproduce

Add a textfield into a dialog body.
Tab to the textfield

## Expected behavior

Focus ring should be fully visible

## Screenshots

Screenshots submitted in corresponding issue https://git.corp.adobe.com/React/react-spectrum-v2/issues/470

## Environment
 - **Spectrum CSS version:** 
 - **Browser(s) and OS(s):** 

## Additional context

Maybe could be solved with negative margins/+padding or padding inside the dialog content area?
overflow-x: visible doesn't help because it can happen at the top and bottom as well.

**Comment by GarthDB** [comment]
We know.

**Comment by snowystinger** [comment]
yeah, I'm not super sure what can be done to improve this, I went through a number of options at the time. We ended up going to containers type elements having the -margins/+padding. I don't think anything new has come up in terms of options. The other thing to note is that we've only covered the ones that are barely bigger than their focused element, when we get to anything with a bigger or more distant focus ring, i don't know what we'll do...

**Comment by pfulton** [comment]
This may be addressed by #1421 - we'll bring this to an upcoming design workshop.
